trino/dbapi.py

```diff
@@ -207,7 +212,7 @@ def _create_request(self):
             self.request_timeout,
         )
 
-    def cursor(self, legacy_primitive_types: bool = None):
+    def cursor(self, legacy_primitive_types: Optional[bool] = None) -> 'Cursor':
```

Comment:
`-> 'Cursor'`? Why no qualify the name?

Reply:
here and other places where we use "strings" as return types

Reply:
Fixed to use [Postponed Evaluation of Annotations](https://peps.python.org/pep-0563/) when type hint contains names that have not been defined yet.